openssl/src/asn1.rs

```diff
@@ -84,77 +109,129 @@ impl fmt::Display for Asn1GeneralizedTimeRef {
     }
 }
 
-/// The type of an ASN.1 value.
-#[derive(Debug, Copy, Clone, PartialEq, Eq)]
-pub struct Asn1Type(c_int);
+/// See ASN.1 specification for the meaning of Asn1TagValues (e.g. https://www.itu.int/en/ITU-T/asn1)
+#[derive(Clone, Copy, Debug, Eq, PartialEq, TryFromPrimitive)]
+#[repr(isize)]
+pub enum Asn1TagValue {
```

Comment:
We don't use enums for these kinds of types: https://github.com/sfackler/rust-openssl/blob/master/openssl/src/rsa.rs#L45-L64

Reply:
Fixed.